src/main/java/uk/gov/hmcts/reform/unspec/callback/CallbackHandler.java

```diff
@@ -9,10 +11,26 @@
 
 public abstract class CallbackHandler {
 
+    public static final String DEFAULT = "default";
+
     protected abstract Map<CallbackType, Callback> callbacks();
 
     public abstract List<CaseEvent> handledEvents();
 
+    public String camundaTaskId() {
+        return DEFAULT;
+    }
+
+    public boolean isEventAlreadyProcessed(CaseData caseData) {
```

Comment:
should we use state flow engine to identify whether event is already fired or not?

Reply:
not sure, this feature is only for camunda processes.

Reply:
should we rename this to avoid confusion then?